.github/workflows/deploy-staging-ec2.yml

```diff
@@ -0,0 +1,34 @@
+on:
+  push:
+    branches:
+      - development
```

Comment:
The trigger for this deployment to staging is very open to debate - I've used a `development` branch that's essentially a feature branch of of `master` to trigger deploys to staging envs in the past, but in a perfect world we might want something that can deploy an ephemeral instance for each pull request in this repository.

Right now this seemed like the best setup given our current infrastructure.